frontend/src/Menu.jsx

```diff
@@ -166,16 +162,19 @@ const Menu = (props) => {
         labelText="Klassen"
         labelIcon={SchoolRoundedIcon}
       >
-        {classList.map((classItem) => (
-          <StyledTreeItem
-            key={classItem.id}
-            nodeId={`/dashboard/classes/${classItem.id}`}
-            labelText={classItem.name}
-            // labelInfo="3"
-            color="#1a73e8"
-            bgColor="#e8f0fe"
-          />
-        ))}
+        {loading ? (
+          <CircularProgress />
+        ) : (
+          classList.map((classItem) => (
+            <StyledTreeItem
+              key={classItem.id}
+              nodeId={`/dashboard/classes/${classItem.id}/insights`}
+              labelText={classItem.name}
+              color="#1a73e8"
+              bgColor="#e8f0fe"
+            />
+          ))
+        )}
```

Comment:
hoort in principe niet bij deze taak, maar kon het niet uitstaan dat het side menu elke keer opnieuw ging laden. Nu is het wat subtieler